{{cookiecutter.project_name}}/.gitignore

```diff
@@ -1,3 +1,2 @@
 tmp/
-*.pyc
-mappings/*.sssom.ttl
```

Comment:
I've been going back and forth on this and welcome any suggestions. My first thought was that the sssom.ttl files were just auto-generated files created as one of the validation checks and so didn't need to be version controlled. Now for C-Path, we are ingesting the sssom.ttl files into the KG so we do need them to be stored in the online git repo. So I thought it might be better to have the default option be to include them in vc (hence the removal of the sssom.ttl files from `.gitignore`). One of my points of confusion regarding the sssom.ttl files is that they are created in the "validate-mappings" step.  Will the `sssom convert` command fail if there is an issue with the sssom.tsv file that doesn't get caught by the `sssom validate` command?

Reply:
I think you are right to allow TTLs in the mapping folder. I would change the makefile that during mapping QC, the TTL files are generated in a temporary directory instead.

I included sssom convert in the validation pipeline because it catches some errors that other approaches cant catch, like illegal prefixing errors, or malformed entity identifiers..

Reply:
I modified the `validate-%` command to output the .ttl files into the tmp directory.